fgpyo/sam/__init__.py

```diff
@@ -547,6 +547,48 @@ def length_on_target(self) -> int:
         """Returns the length of the alignment on the target sequence."""
         return sum([elem.length_on_target for elem in self.elements])
 
+    def get_alignment_offsets(self, reverse: bool = False) -> Tuple[int, int]:
+        """
+            Get the starting and ending offsets for the alignment based on the CIGAR string.
+
+            Args:
+                reverse: If True, count from the end of the read sequence.
+                    Otherwise (default behavior), count from the beginning of the read sequence.
```

Comment:
This is unclear to me - does it _just_ count from the end?  Or does it also swap the order of "start" vs. "end"?  I think you're saying that:

```
cigar.get_alignment_offsets(reverse=True) == cigar.reverse.get_alignment_offsets()
```
?

Reply:
yes.